contracts/token/oft/v2/precrime/IOFTV2View.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0;
+
+interface IOFTV2View {
+    /// @notice simulates receiving of a message
+    function lzReceive(uint16 _srcChainId, bytes32 _scrAddress, bytes memory _payload, uint _totalSupply) external view returns (uint);
+
+    function getInboundNonce(uint16 _srcChainId, bytes32 _scrAddress) external view returns (uint64);
+
+    /// @notice returns the total supply for OFTV2 or outbound amount for ProxyOFTV2
+    function getTotalSupply() external view returns (uint);
```

Comment:
this is little confusing, should rename this to getCurrentState()

Reply:
Done